src/resources/Microsoft.Authorization/policyDefinitions/Deploy-Private DNS zone-PaaS-Private endpoint-Allowed values.json

```diff
@@ -0,0 +1,233 @@
+    {
+    "name": "Configure a private DNS Zone ID for Paas services-Generic policy for PaaS services dosent has builtin policies",
```

Comment:
Please shorten the policy name and filename, and ensure filename and policy "name" are the same.